### ISSUE TITLE
YAML vs. JSON

### ISSUE THREAD

**Issue by rettinghaus** [citation-file-format/citation-file-format#431]
The changelog for version 1.2 states:

> switched from YAML schema to JSON schema

Yet all the examples here and in the documentation are still YAML files. I'm confused.

**Comment by kevinmatthes** [comment]
@rettinghaus

CFF is a subset of YAML.  In order to define the rules when a CFF can be considered "valid", the CFF team decided to use a certain Python library.  This library requires the definition of a JSON file which contains the rules for CFF.  Validators like `cffconvert` read the JSON manifest, named `schema.json`, as well as the CITATION.cff to validate and try to match the read CFF file against the JSON schema.

In summary:  JSON is CFF's language kernel while YAML is the representation.

Did this answer your question?

**Comment by rettinghaus** [comment]
Yes, thank you! Is this somewhere explained in the documentation? Couldn't find it.

**Comment by kevinmatthes** [comment]
@rettinghaus

Yes, it is:  https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md?plain=1#L8

**Comment by rettinghaus** [comment]
Great! But there is no link in the README. Please add one for blind guys like me ;-).

**Comment by kevinmatthes** [comment]
@rettinghaus

I looked through the README and found such a link in line 46:  https://github.com/citation-file-format/citation-file-format/blob/main/README.md?plain=1#L46

It points to the schema guide in general as this is the main documentation for CFF (from my point of view).

Would this suffice or should the description be expanded?

**Comment by rettinghaus** [comment]
Maybe just add another sentence that CFF is specified with JSON?

**Comment by kevinmatthes** [comment]
@rettinghaus

Okay, I will suggest a general enhancement regarding the schema guide to the CFF team.  I noticed some related issues, so I guess that the description of / redirection to the schema guide should be enhanced in general.
